src/test/java/uk/gov/justice/hmpps/prison/dsl/Alias.kt

```diff
@@ -25,6 +25,7 @@ class AliasBuilderRepository(
     offenderRepository.save(
       Offender.builder()
         .nomsId(it.nomsId)
+        .rootOffenderId(it.rootOffenderId)
```

Comment:
the `rootOffender` is mapped as `insertable=false` and `updateable=false`, the mapping is handled by the `rootOffenderId` here instead so needs to be set